### PR TITLE
`.christmasify` Command

### LINKED ISSUE #937: New ".christmasify" command
## Description
Changes a users current profile picture into something more seasonal, similar to commands like .spookify

## Reasoning
It would be fun and spread the holiday cheer

## Proposed Implementation
Adding Christmas hats, snowflakes, etc etc 

## Would you like to implement this yourself?
- [ ] I'd like to implement this feature myself
- [x] Anyone can implement this feature

I would do it myself, but I'm rather busy and don't know how to use pillow, as I've never experimented with it before

### PR DESCRIPTION
**Pull request by DMFriends**
## Relevant Issues

Closes #937 

## Description

I created a new `.christmasify` cog in the `avatar_modify.py` file. I made this command very similar to the currently existing `.spookify` command, with different images to represent Christmas / New Years.

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] Read all the comments in this template?
- [x] Ensure there is an issue open, or link relevant discord discussions?
- [x] Read and agree to the [contributing guidelines](https://pythondiscord.com/pages/contributing/contributing-guidelines/)?

### CODE REVIEW

File: bot/exts/avatar_modification/avatar_modify.py [hunk 1]
```diff
@@ -316,6 +317,39 @@ async def spookyavatar(self, ctx: commands.Context) -> None:
 
             await ctx.send(file=file, embed=embed)
 
+    @avatar_modify.command(
+        aliases=("christmasify", "cavatar"),
+        root_aliases=("christmasavatar", "christmasify", "cavatar"),
+        brief="Christmasify a user's avatar."
```
Comment (onerandomusername) [inline bot/exts/avatar_modification/avatar_modify.py:323]: your own* avatar

File: bot/utils/christmas/christmasifications.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    Adds a snowflake to the image.
+
+    The snowflake is of a size at least one-fifths that of the original image and may be rotated
+    up to 90 degrees anti-clockwise.
+    """
+    im = im.convert("RGBA")
+    wt, ht = im.size
+    snowflake = Image.open("bot/resources/holidays/christmas/snowflake.png").convert("RGBA")
+    snowflake_size = randint(wt // 10, wt // 7)
+    rot = randint(0, 90)
```
Comment (burakhanaksoy) [inline bot/utils/christmas/christmasifications.py:37]: might not need a new variable as rot, we can use as 

```py
snowflake = snowflake.rotate(randint(0,90))
```

Comment (DMFriends) [reply bot/utils/christmas/christmasifications.py:37]: Good suggestion, I agree. Will fix.

**Review by onerandomusername — CHANGES_REQUESTED** [review]
haven't tested much, other than before the asset change.

File: bot/exts/avatar_modification/avatar_modify.py [hunk 3]
```diff
@@ -316,6 +317,39 @@ async def spookyavatar(self, ctx: commands.Context) -> None:
...
+            file = await in_executor(
+                PfpEffects.apply_effect,
+                image_bytes,
+                christmasifications.get_random_effect,
+                file_name
+            )
+
+            embed = discord.Embed(
+                title="Is this you or am I just really paranoid?",
+                colour=Colours.soft_red
```
[the remark this answers is not in the document]
Comment (DMFriends) [reply bot/exts/avatar_modification/avatar_modify.py:346]: Oh yeah lol, I still have a lot to work on as you can see :). Will get there.

[the remark this answers is not in the document]
Comment (DMFriends) [reply bot/exts/avatar_modification/avatar_modify.py]: I just pushed a change @onerandomusername, are you satisfied now? :)

[the remark this answers is not in the document]
Comment (onerandomusername) [reply bot/exts/avatar_modification/avatar_modify.py]: The title is more of a halloween feel still, and not really christmasy

[the remark this answers is not in the document]
Comment (DMFriends) [reply bot/exts/avatar_modification/avatar_modify.py]: Oh, I thought you were pointing at the colors, mb. I'll fix the title too

**Comment by DMFriends** [comment]
Quick update on this: I'm feeling like I won't be able to work on this simply because I can't seem to understand Pillow. If anyone else wants to pick this up, feel free, just be sure to get assigned on the issue #937 first!

**Pull request closed without merging**